libs/ui/src/lib/table-responsive/table-responsive.component.html

```diff
@@ -0,0 +1,60 @@
+<cvi-ng-table class="cvi-table-responsive__cvi-table" [data]="data" >
+    <ng-template #headers>
+        <ng-container *ngFor="let headerLabel of headerLabels">
+            <th cvi-ng-header-cell>{{ headerLabel }}</th>
+        </ng-container>
+        <th cvi-ng-header-cell>
+            <cvi-ng-screenreader-text label="Icons"></cvi-ng-screenreader-text>
+        </th>
+    </ng-template>
+
+    <ng-template #rows let-row>
```

Comment:
Can I somehow rework this code, so I don't need to hard code it in table-responsive html? So table-responsive pass template to table component. Same with card component.

Reply:
Yes, ideally it should not be hard coded :) This needs a bit of thinking, but common templates, defined once, serving same content to both card and table should be the way to go.

Reply:
Should be resolved now - contents of both header and body cells is fully customisable.